app/assets/stylesheets/reservations/_index.css.scss

```diff
@@ -3,3 +3,7 @@
   border-radius: 5px;
   margin-bottom: 20px;
 }
+
+#reservations-list {
+  height: 10em;
```

Comment:
Fair enough, much appreciated :smile:. Does that mean this one gets your stamp of approval? Also, do you mind checking out my [comment](https://github.com/YaleSTC/reservations/issues/1288#issuecomment-157267627) in #1288 and seeing if what I concluded makes sense? Thanks!